### DISCUSSION TITLE
Docker container does not respect SIGTERM signal for clean shutdown

### DISCUSSION THREAD

**Discussion by jpawlowski** [requarks/wiki#5413] [Error / Bug Report] [answered]
Hello,

currently one cannot shutdown Docker service smoothly without manually killing the Wiki.js container before.
It seems the Wiki.js container image does not consider to work with SIGTERM signals to shut down the NodeJS server and end the startup entry script.

The image version I tried this is `requarks/wiki:2`.

Would the maintainer consider improving the image in this regard?

**Comment by NGPixel** [comment, marked as answer]
Added in 0425b82c833aeaccd49532dd725602912b3c4346

Will be part of the next release.

**Reply by NGPixel** [reply]
Now available in 2.5.285
